release-notes.md

```diff
@@ -374,6 +381,66 @@
     - [Unit testing](#unit-testing)
     - [Integration tests parallelization](#integration-tests-parallelization)
 
+# v47.0
+
+This release has been tracked in [v47.0
+group](https://github.com/orgs/cloud-hypervisor/projects/6/views/4?filterQuery=release%3A%22Release+47%22)
+of our [roadmap project](https://github.com/orgs/cloud-hypervisor/projects/6/).
+
+### Block Device Error Reporting to the Guest
+
+Instead of exiting on I/O errors, the `virtio-block` device now reports
+errors to the guest using `VIRTIO_BLK_S_IOERR`. It improves user
+experience particularly when the guest rootfs is not backed by the
+affected block device. (#7107)
+
+### Nice Error Messages on Exit
+
+We now have the chain of errors being reported and printed nicely, when
+Cloud Hypervisor or ch-remote exits on errors. (#7066)
+
+### Alphabetically Sorted CLI Options for ch-remote
+
+To improve readability, ch-remote now prints help information in
+alphabetical order. (#7130)
+
+### Notable Bug Fixes
+
+* Error out early when block device serial is too long (#7124)
+* Fix partial commands being discarded for `virtio-vsock` (#7195)
+* Disable the broken interrupt support for the `rtc_pl031` device to
+  prevent spurious guest interrupts (#7199)
+
+### Deprecations
+
+* A default IP (`192.168.249.1`) and mask (`255.255.255.0`) now are
```

Comment:
s/now are/are currently/

The "now" makes it sound like a change to this behaviour.